### DISCUSSION TITLE
Upload Floorp web browser on the Microsoft Store

### DISCUSSION THREAD

**Discussion by MuerteRoja** [Floorp-Projects/Floorp#1169] [Feature Request]
Hello, good evening, some time ago on the Codeberg website I saw that Malte Jurgens, one of the developers of the Librewolf web browser, has automated the updates of said browser in the Microsoft Store, Librewolf is a web browser similar to Floorp, since both are forks based on Firefox and also advocate for user security and privacy, currently Librewolf has been uploaded by this same developer in the Microsoft Store for a long time (although you also have it available on the browser's official website) I think it would be good, and It would be quite practical if they did the same with the Floorp web browser, in this way you save several tasks, since when you click on the “Get updates” button, in addition to updating the web browser, you would also be updating the other applications that you have installed on your computer from the same Microsoft Store, I understand that this would be a complicated job if there were only 2 or 3 developers maintaining Floorp, however, from what I understand, Floorp is a web browser which It is receiving maintenance from several developers, therefore, I think that uploading the Floorp browser to the Microsoft Store is possible.
Here I leave the link to the forum where they talk about the Librewolf issue in the Microsoft Store: https://codeberg.org/librewolf/issues/issues/867
And here I leave the link of the process in Codeberg for automating updates: https://codeberg.org/librewolf/bsys6/commit/fd979fc04eb3993aec442c7b04c195bb3ca3d059
![Captura de pantalla 2024-05-21 215655](https://github.com/Floorp-Projects/Floorp/assets/126993006/309f370d-f77f-4514-8ad2-dd5a31e4366c)

**Comment by creeper-0910** [comment]
In order to upload Floorp to the Microsoft Store, you need to disable one part of the functionality and build it again, so
We do not plan to support this at this time.

**Reply by thomassth** [reply]
What part of functionality would that be? Just curious on what's the limitations here

**Reply by creeper-0910** [reply]
The default browser settings are required to be disabled at the time of release in the MS Store.
